docs/docs/reference/02-manual-installation-launch.md

```diff
@@ -0,0 +1,311 @@
+---
+sidebar_position: 1
+title: Manual Installation and Launch
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The following sections explain how the installation and launch steps of the AICA Launcher can be performed manually.
+The pre-requisites are still a valid license and a host with Docker installed. For the rest of this guide, it will be
+assumed that a valid license has been saved to a file called `aica-license.toml` on the host machine.
+
+## Logging in to the AICA package registry
+
+To authenticate docker to login and pull images from the registry, run the following command (no replacement of
+`USERNAME` required):
+
+```shell
+cat aica-license.toml | docker login registry.licensing.aica.tech -u USERNAME --password-stdin
+```
+
+## Configuring AICA packages with a manifest file
+
+A runtime application image is configured using a simple **manifest file** defining the version of the `core` package
+to use and optionally defining additional add-on packages. The manifest file contains a custom docker syntax header
+pointing to AICA's app-builder tool, and the `docker build` command is used to bundle all listed packages into a final
+runtime image.
+
+### Configuring a minimal runtime image with the `core` package
+
+The manifest file must contain a syntax header and a list of packages. The minimal version of the manifest includes
+only the `core` package. The version of the `core` package can be changed according to the latest release.
+
+:::info
+
+In the past, you might have seen applications using the `aica-package.toml` filename. While you can use any filename as
+we do not enforce any, we recommend using `aica-application.toml` to avoid confusion with the `aica-package.toml` file
+which is used for building packages using `package-builder`.
+
+:::
+
+```toml title="aica-application.toml"
+#syntax=ghcr.io/aica-technology/app-builder:v2
```

Comment:
```suggestion
#syntax=ghcr.io/aica-technology/app-builder:v2.0.4
```
not sure if we need to go with the minor version. Just adding the comment as a note